apps/web/components/booking/BookingListItem.tsx

```diff
@@ -508,72 +511,94 @@ function BookingListItem(booking: BookingItemProps) {
           </DialogFooter>
         </DialogContent>
       </Dialog>
-      <tr data-testid="booking-item" className="hover:bg-muted group flex flex-col transition sm:flex-row">
```

Comment:
All the booking badges were under one flex box column with the booking details. This caused ugly rendering when we're adding more badges.

Comparing where the badges are
### Before
![CleanShot 2024-12-08 at 21 33 35@2x](https://github.com/user-attachments/assets/6b4d34bb-9d24-4135-b776-8d755c96c2a3)


### After
![CleanShot 2024-12-08 at 21 34 20@2x](https://github.com/user-attachments/assets/ed7d4851-3b55-45bf-be40-969e51c36eb7)